01/app.js

```diff
@@ -0,0 +1,29 @@
+const form = document.querySelector("form");
+const usersList = document.querySelector(".users-list");
```

Comment:
Uważałbym na nazewnictwo. Dodałbym, "Element" tj. `userListElement`. W pierwszej chwili myślałem, że to tablica.